update_images_test

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
```

Comment:
What about refactoring the existing script into a common bash function and then call that function both from `update_images` & `update_images_test` by passing some params for beta/actual images ?

Also, maybe `update_images_test` -> `update_test_images`.

Reply:
Removed it and performed it as you suggest